client/client.go

```diff
@@ -450,52 +450,55 @@ func (c *client) processTSORequests(stream pdpb.PD_TsoClient, dcLocation string,
 
 	if err := stream.Send(req); err != nil {
 		err = errors.WithStack(err)
-		c.finishTSORequest(requests, 0, 0, err)
+		c.finishTSORequest(requests, 0, 0, 0, err)
 		return err
 	}
 	resp, err := stream.Recv()
 	if err != nil {
 		err = errors.WithStack(err)
-		c.finishTSORequest(requests, 0, 0, err)
+		c.finishTSORequest(requests, 0, 0, 0, err)
 		return err
 	}
 	requestDurationTSO.Observe(time.Since(start).Seconds())
 	tsoBatchSize.Observe(float64(count))
 
 	if resp.GetCount() != uint32(len(requests)) {
 		err = errors.WithStack(errTSOLength)
-		c.finishTSORequest(requests, 0, 0, err)
+		c.finishTSORequest(requests, 0, 0, 0, err)
 		return err
 	}
 
-	physical, logical := resp.GetTimestamp().GetPhysical(), resp.GetTimestamp().GetLogical()
+	physical, logical, suffixBits := resp.GetTimestamp().GetPhysical(), resp.GetTimestamp().GetLogical(), resp.GetTimestamp().GetSuffixBits()
```

Comment:
I think pass the `suffixBits` is too implementation specific, what about pass the sign itself?

Reply:
I don't think it's necessary, for the client user, all they get for the TSO result is two `int64` integers, and the `suffixBits` here is rather considered as internal computing usage only. We should make it easy to implement the shift operation.